packages/admin/resources/views/components/layouts/app/sidebar/item.blade.php

```diff
@@ -48,16 +49,35 @@
             </span>
 
             @if (filled($badge))
-                <span
-                    @class([
-                        'inline-flex items-center justify-center ml-auto rtl:ml-0 rtl:mr-auto min-h-4 px-2 py-0.5 text-xs font-medium tracking-tight rounded-xl whitespace-normal',
-                        'text-primary-700 bg-primary-500/20' => ! $active,
-                        'text-white bg-white/20' => $active,
-                        'dark:text-primary-500' => (! $active) && config('filament.dark_mode'),
-                    ])
-                >
-                    {{ $badge }}
-                </span>
+                <span @class([
+                    'inline-flex items-center justify-center ml-auto rtl:ml-0 rtl:mr-auto min-h-4 px-2 py-0.5 text-xs font-medium tracking-tight rounded-xl whitespace-normal',
+                    'text-white bg-white/20' => $active,
+                    ...match ($badgeColor ?? null) {
+                        'danger' => [
+                            'text-danger-700 bg-danger-500/10' => ! $active,
```

Comment:
I feel like we should refactor this and move `! $active` outside the match expression.

Reply:
No, because each `! $active` is color-specific

Reply:
How do you mean? They all have `! $active`, so it could be extracted, right?